src/downshift.js

```diff
@@ -67,7 +68,12 @@ class Downshift extends Component {
     defaultIsOpen: false,
     getA11yStatusMessage,
     id: generateId('downshift'),
-    itemToString: i => (i == null ? '' : String(i)),
+    itemToString: i => {
```

Comment:
I'd like to have a test for this. Could you add one in [`downshift.misc.js`](https://github.com/paypal/downshift/blob/master/src/__tests__/downshift.misc.js)? Just do `jest.spyOn(console, 'warn')` on the first line of the test, then `console.warn.mockRestore()` on the last line to clean up. You could even use a snapshot if you want: `expect(console.warn).toHaveBeenCalledTimes(1)` and then `expect(console.warn.mock.calls).toMatchSnapshot()`

Reply:
Sure!